profiles/NIST_SP-800-53_rev5_MODERATE-baseline_profile.json

```diff
@@ -1,373 +1,359 @@
-                "ra-5.2",
-                "ra-5.5",
-                "ra-5.11",
-                "ra-7",
-                "ra-9",
-                "sa-1",
-                "sa-2",
-                "sa-3",
-                "sa-4",
-                "sa-4.1",
-                "sa-4.2",
-                "sa-4.9",
-                "sa-4.10",
-                "sa-5",
-                "sa-8",
-                "sa-9",
-                "sa-9.2",
-                "sa-10",
-                "sa-11",
-                "sa-15",
-                "sa-15.3",
-                "sa-22",
-                "sc-1",
-                "sc-2",
-                "sc-4",
-                "sc-5",
-                "sc-7",
-                "sc-7.3",
-                "sc-7.4",
-                "sc-7.5",
-                "sc-7.7",
-                "sc-7.8",
-                "sc-8",
-                "sc-8.1",
-                "sc-10",
-                "sc-12",
-                "sc-13",
-                "sc-15",
-                "sc-17",
-                "sc-18",
-                "sc-20",
-                "sc-21",
-                "sc-22",
-                "sc-23",
-                "sc-28",
-                "sc-28.1",
-                "sc-39",
-                "si-1",
-                "si-2",
-                "si-2.2",
-                "si-3",
-                "si-4",
-                "si-4.2",
-                "si-4.4",
-                "si-4.5",
-                "si-5",
-                "si-7",
-                "si-7.1",
-                "si-7.7",
-                "si-8",
-                "si-8.2",
-                "si-10",
-                "si-11",
-                "si-12",
-                "si-16",
-                "sr-1",
-                "sr-2",
-                "sr-2.1",
-                "sr-3",
-                "sr-5",
-                "sr-6",
-                "sr-8",
-                "sr-10",
-                "sr-11",
-                "sr-11.1",
-                "sr-11.2",
-                "sr-12"
-              ]
+              "addr-lines": [
+                "National Institute of Standards and Technology",
+                "Attn: Computer Security Division",
+                "Information Technology Laboratory",
+                "100 Bureau Drive (Mail Stop 8930)"
+              ],
+              "city": "Gaithersburg",
+              "state": "MD",
+              "postal-code": "20899-8930"
             }
           ]
         }
       ],
-      "merge": {
-        "as-is": true
-      },
-      "back-matter": {
-        "resources": [
+      "responsible-parties": [
+        {
+          "role-id": "creator",
+          "party-uuids": [
+            "cde369ce-57f8-4ec1-847f-2681a9a881e7"
+          ]
+        },
+        {
+          "role-id": "contact",
+          "party-uuids": [
+            "cde369ce-57f8-4ec1-847f-2681a9a881e7"
+          ]
+        }
+      ]
+    },
+    "imports": [
+      {
+        "href": "NIST_SP-800-53_rev5_catalog.json",
```

Comment:
When loading https://raw.githubusercontent.com/EasyDynamics/oscal-demo-content/c2f9358221da81a280f6e2ff00f7ea2077e6ef26/profiles/NIST_SP-800-53_rev5_MODERATE-baseline_profile.json into the viewer, I am getting a 404 error for: `https://raw.githubusercontent.com/GSA/fedramp-automation/master/baselines/rev4/json/FedRAMP_rev4_HIGH-baseline_profile.json`

Again the previous version we had https://raw.githubusercontent.com/EasyDynamics/oscal-demo-content/main/profiles/NIST_SP-800-53_rev5_MODERATE-baseline_profile.json loads correctly. 

I want to blame the `href` but may be completely wrong.

Reply:
the href for this one is incorrect, i have resolved this one